server/data/nomisClientBuilder.ts

```diff
@@ -191,7 +191,7 @@ export = (token) => {
       return oauthRestClient.getResource(`/api/user/me`)
     },
 
-    getUserRoles(): Promise<Role> {
+    getUserRoles(): Promise<[Role]> {
```

Comment:
this is weird, is this a valid way to define array tyoes?
rather than `Role[]` or `Array<Role>`